hub/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .api.dataset import Dataset
 
-__version__ = "2.0.0"
+__version__ = "2.0a0"
```

Comment:
FYI https://www.python.org/dev/peps/pep-0440/#implicit-pre-release-number

Reply:
@benchislett not sure if I understand the issue here?

`Pre releases allow omitting the numeral in which case it is implicitly assumed to be 0. The normal form for this is to include the 0 explicitly. This allows versions such as 1.2a which is normalized to 1.2a0.`

I explicitly added `a0` instead of `a`, because we are going to iterate `a1`, `a2`, `a3`, ... for alpha releases.